src/particle/particle/convert.py

```diff
@@ -109,6 +109,9 @@ def filter_file(fileobject):
         if not line.lstrip("\ufeff").lstrip().startswith("*"):
             stream.write(line)
     stream.seek(0)
+
+    fileobject.close()
```

Comment:
Shouldn't you only close it if you opened it?

Reply:
Absolutely. It is a possibility to call the method with a file opened. I will add a check.